hack/functests.sh

```diff
@@ -24,6 +24,7 @@ DOCKER_TAG_ALT=${DOCKER_TAG_ALT:-devel_alt}
 KUBEVIRT_E2E_PARALLEL_NODES=${KUBEVIRT_E2E_PARALLEL_NODES:-3}
 KUBEVIRT_FUNC_TEST_GINKGO_ARGS=${FUNC_TEST_ARGS:-${KUBEVIRT_FUNC_TEST_GINKGO_ARGS}}
 KUBEVIRT_FUNC_TEST_LABEL_FILTER=${FUNC_TEST_LABEL_FILTER:-${KUBEVIRT_FUNC_TEST_LABEL_FILTER}}
+KUBEVIRT_FUNC_TEST_GINKGO_TIMEOUT=${KUBEVIRT_FUNC_TEST_GINKGO_TIMEOUT:-4h}
```

Comment:
I think we can go back to the 3 hour timeout after the recent lane runtime reductions. What do you think?

Reply:
If lanes often get stuck "forever" sure, but if that's a rare occurrence I'd rather have a very long timeout, just in case. Nothing worse that seeing a lane that could have succeeded getting killed by timeout...

Reply:
I was also thinking about being able to adjust the timeout for lanes that need less time so that they will timeout earlier - this way we wouldn't waste three hours on lanes that get stuck while they usually complete with an hour.

Reply:
> If lanes often get stuck "forever" sure, but if that's a rare occurrence I'd rather have a very long timeout, just in case. Nothing worse that seeing a lane that could have succeeded getting killed by timeout...

@jean-edouard I agree.

So do you think the timeout of 4h is still not sufficient, since it was 3h before?

Reply:
If we're somewhat confident all lanes will finish in 3 hours, then adding an hour seems like a comfortable margin.

Reply:
The e2e tests dashboard shows a maximum runtime of a bit more than 2h currently:

[![image](https://github.com/user-attachments/assets/eb27035b-56b2-4b2e-957b-5a99d97e801d)](https://grafana.ci.kubevirt.io/d/gY4hNc5Iz/kubevirt-kubevirt-e2e-test-runtimes?orgId=1&from=now-24h&to=now)